src/nvim/testdir/test_eval_stuff.vim

```diff
@@ -117,6 +117,22 @@ func Test_skip_after_throw()
   endtry
 endfunc
 
+" scriptversion 1
```

Comment:
`:scriptversion` tests come after `Test_let_errmsg()`.

```vim
scriptversion 1
func Test_vvar_scriptversion1()
  call assert_equal(15, 017)
  call assert_equal(15, 0o17)
  call assert_equal(15, 0O17)
  call assert_equal(18, 018)
  call assert_equal(511, 0o777)
endfunc
``` 
is missing.

Reply:
I'll add it. :+1:

Reply:
A few of the functions in `test_eval_stuff.vim` seems to be in a different order compared to Vim.
I could re-arrange them all if you'd like to match Vim?

Reply:
Sure in a separate commit.